runtime/lua/vim/_defaults.lua

```diff
@@ -147,6 +147,22 @@ do
       require('vim._comment').textobject()
     end
     vim.keymap.set({ 'o' }, 'gc', textobject_rhs, { desc = 'Comment textobject' })
+
+    local insert_rhs = function()
+      return require('vim._comment').insert()
```

Comment:
`vim._comment` seems to be not public so user cannot rely on it if wants to map `insert` to another shortcut.

Reply:
If this PR is going to trigger requested for a public interface, let's just close this PR.

Reply:
People are not mindreaders. It was probably put there because most other files are there. Why does `_default` live in `vim` for example?

Reply:
> I'm actually not sure why `_comment` lives on `vim.`.

I thought it was the common way to place the "private" Lua code. I used `vim._system` as a reference. If it is better be moved - no problem.